source/object/Display.es6

```diff
@@ -0,0 +1,42 @@
+import * as PIXI from 'pixi.js';
```

Comment:
I'd like to follow a MVVM-based widget model, so we should maybe call this MapView.es6, or LevelView.es6.
Also don't want to hardcode 800x600 pixels, but that's fine for right now.

Reply:
Right, we could have this as a base class, then have a widget model, makes sense.